.claude/commands/kb-context.md

```
@@ -0,0 +1,71 @@
+  - mcp__gitkb__kb_context
+  - mcp__gitkb__kb_status
+  - mcp__gitkb__kb_list
+  - mcp__gitkb__kb_show
+  - mcp__gitkb__kb_checkout
+  - mcp__gitkb__kb_create
+  - mcp__gitkb__kb_commit
+  - Bash(git kb:*)
+description: Load and validate project context, bootstrapping if needed
+---
+
+Load project context following the AGENTS.md PATH A/B/C flow.
```

markdownlint-cli2 (0.20.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)